integr8ly/integreatly-operator
# MGDAPI-90 Add customer facing Grafana to managed-api

# Description
Added grafana operator and instance to managed-api install type

To verify: 
1, Run managed-api install. 
2, Confirm redhat-rhmi-customer-monitoring-operator product namespace is created.
3. Confirm grafana-operator pod exists and is ready
4. Confirm grafana-deployment pod exists and is ready
5. Confirm grafana route exists and works. 
6. Login to grafana instance using kubeadmin or dedicated-admin user. 

## Type of change
